Command/MigrationsLatestDoctrineCommand.php

```diff
@@ -35,6 +35,7 @@ protected function configure()
             ->setName('doctrine:migrations:latest')
             ->addOption('db', null, InputOption::VALUE_REQUIRED, 'The database connection to use for this command.')
             ->addOption('em', null, InputOption::VALUE_REQUIRED, 'The entity manager to use for this command.')
+            ->addOption('shard', null, InputOption::VALUE_OPTIONAL, 'The shard connection to use for this command.')
```

Comment:
Yes it is: https://github.com/vincentchalamon/DoctrineMigrationsBundle/blob/sharding/Command/Helper/DoctrineCommandHelper.php#L44